lib/heroku/command/pg.rb

```diff
@@ -85,16 +85,29 @@ def promote
     end
     validate_arguments!
 
+    db = db.sub(/_URL$/, '') # allow promoting with a var name
     addon = resolve_addon!(db)
 
-    attachment_name = 'DATABASE'
-    action "Promoting #{addon['name']} to #{attachment_name}_URL on #{app}" do
+    promoted_name = 'DATABASE'
+
+    action "Ensuring an alternate alias for existing #{promoted_name}" do
```

Comment:
This is kind of misleading and potentially confusing in the (common?) case when there is no existing attachment, no?